src/widgets.ts

```diff
@@ -83,6 +85,33 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with iframe widget about navigation events
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget, which if it's
+   * listening for the 'mx/navigation' event, it can make its changes and then
+   * send its own post message event back to the SDK for the 'onNavigation' callback
+   */
+  navigateBack() {
+    const iframeElement = this.iframe.contentWindow
+    const data = { mx: true, type: "mx/navigation", payload: { action: "back" } }
+
+    const baseUrlPattern = /^https?:\/\/[^/]+/i
+    let widgetBaseUrl
+
+    if (this.options.url && this.options.url.match(baseUrlPattern)) {
+      widgetBaseUrl = this.options.url.match(baseUrlPattern)?.[0]
+    } else if (this.options.proxy && this.options.proxy.match(baseUrlPattern)) {
```

Comment:
We don't load `this.options.proxy` in the iframe, it's only used when we make the request to get the SSO URL.